source/plugin/internals/implementation.rst

```diff
@@ -46,7 +46,7 @@ Example Using Gradle
 .. code-block:: groovy
 
     dependencies {
-        compile 'org.spongepowered:spongevanilla:1.12.2-7.1.0-BETA-116:dev'
+        compile 'org.spongepowered:spongevanilla:1.12.2-7.1.0-RC116:dev'
```

Comment:
No, this won't compile.

The version change happened with RC123. 
There's not spongevanilla:1.12.2-7.1.0-RC116.

See https://www.spongepowered.org/downloads/spongevanilla/stable/1.12.2

Reply:
Corrected